### DISCUSSION TITLE
How to make my task output "clickable" in the UI

### DISCUSSION THREAD

**Discussion by funkypenguin** [concourse/concourse#7213] [Help & Support]
Hey folks!

Our Concourse tasks produce a _lot_ of output, and to make it easier to parse afterwards, I'm sending it all to a Grafana Loki instance, and providing the user with a dashboard URL.

Currently I just echo the dashboard URL in my task, and the user manually copies/pastes it into his browser. Is it possible to output a "clickable" link in a task output?

Thanks!
D

**Comment by HemangLavana** [comment]
I would like to see this feature as well. If concourse could parse and make all URLs in the output "clickable" link, that would be very helpful. I have seen this happen in certain contexts such as "get" output of a job run but not in task output (which would require parsing), nor in resource check context where 'url' metadata is clearly identified.
